backend/src/node.rs

```diff
@@ -33,16 +33,20 @@ pub struct Node {
     location: Option<Arc<NodeLocation>>,
     /// Flag marking if the node is stale (not syncing or producing blocks)
     stale: bool,
-    /// Connected at timestamp
-    connected_at: Timestamp,
+    /// Unix timestamp for when node started up (falls back to connection time)
```

Comment:
Should we really be falling back to connection time here? They are different things. Would switching to an `Option<Timestamp>` be very messy?
If we're ever interested in the time the node connected – and I think we're not – I think it should be a separate field.

Reply:
It's a fallback for older version which didn't send the field, which is also what the current functionality is. I believe at this point most nodes should be sending this over, to turn it into an `Option<Timestamp>` requires bumping protocol version at connection handshake with FE, and some changes there to visually account for `null`s.

Reply:
I see. Can we fall back to `UNIX_EPOCH` or some other value "zero-ish" value?

Reply:
This shows up as Uptime in the FE, I don't think showing 50 years of uptime is better than showing the time the node connected to telemetry at as fallback :)